src/Parser/VengeanceDemonHunter/Modules/Statistics/Spells/EmpowerWards.js

```diff
@@ -21,7 +26,7 @@ class EmpowerWards extends Module {
         icon={<Icon icon="ability_demonhunter_empowerwards" alt="Empower Wards" />}
```

Comment:
Please use `SpellIcon` here to users can consistently hover the icon for tooltips about the spells to learn more about them.